main.py

```diff
@@ -1,7 +1,46 @@
-import init_django_orm  # noqa: F401
-
+import init_django_orm  # noqa: F401p
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [("Western",), ("Action",), ("Dramma",)]
+    for genre_name, in genres:
+        Genre.objects.create(name=genre_name)
+        Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
The update operation for changing 'Dramma' to 'Drama' is placed inside the loop for creating genres. This will result in multiple unnecessary update queries. Consider moving this line outside the loop.